pyproject.toml

```diff
@@ -18,7 +18,13 @@ features = ["pyo3/extension-module"]
 module-name = "scylla._rust"
 
 [dependency-groups]
-dev = ["pytest", "pytest-asyncio","maturin"]
+dev = [
+    "pytest",
+    "pytest-asyncio",
+    "maturin",
+    "ruff>=0.14.5",
+    "basedpyright>=1.33.0",
+]
```

Comment:
Ah, the tools are added in the second commit. Could you move this change to the first commit?

Reply:
Unresolving. I don't see answer to my question about version restrictions. In other words, why `ruff>=0.14.5` and not just `ruff`?